src/sflow/hsflowd/patch/dropmon/0001-sflow-enabled-drop-monitor-support-for-SONiC.patch

```diff
@@ -3,47 +3,27 @@ From: Vadym Hlushko <vadymh@nvidia.com>
 Date: Tue, 25 Jan 2022 12:59:40 +0000
 Subject: [PATCH] [sflow] enabled drop monitor support for SONiC
 
-Signed-off-by: Vadym Hlushko <vadymh@nvidia.com>
----
- src/Linux/Makefile                   | 2 +-
- src/Linux/hsflowd.c                  | 8 ++++++++
- src/Linux/scripts/hsflowd.conf.sonic | 2 ++
- 3 files changed, 11 insertions(+), 1 deletion(-)
-
-diff --git a/src/Linux/Makefile b/src/Linux/Makefile
-index 8128cf2..cf538e7 100644
---- a/src/Linux/Makefile
-+++ b/src/Linux/Makefile
-@@ -34,7 +34,7 @@ FEATURES_DENT= DENT PSAMPLE SYSTEMD DROPMON
- FEATURES_EOS= EAPI
- FEATURES_OS10= OS10 DBUS SYSTEMD
- FEATURES_OPX= OPX DBUS SYSTEMD
--FEATURES_SONIC= SONIC PSAMPLE DOCKER
-+FEATURES_SONIC= SONIC PSAMPLE DOCKER DROPMON
```

Comment:
Why is `DROPMON` removed here? Is it no longer needed?

Reply:
@saiarcot895  This patch is removed, because its already part of 2.0.45 version
https://github.com/sflow/host-sflow/blob/0de668dd9e7a19ceef2dfc22ec46485fe829498b/src/Linux/Makefile#L49